docs/core/testing/unit-testing-mstest-runner-intro.md

```diff
@@ -0,0 +1,253 @@
+---
+title: MSTest Test Runner
+description: Learn about MSTest Test Runner, a lightweight way to run tests without depending on dotnet SDK.
+author: nohwnd
+ms.author: jajares
+ms.date: 11/28/2023
+ms.custom: 
+---
+
+# Overview
+
+MSTestRunner is a light-weight and portable alternative to [microsoft/vstest](https://github.com/microsoft/vstest) for running tests in continuous integration pipelines, and in VisualStudio TestExplorer (Coming soon!).
+
+This runner is embedded directly in your MSTest test project, and there is no additional application (e.g. vstest.console / dotnet test) or any additional infrastructure (e.g. dotnet SDK) needed to run your tests.
+
+MSTestRunner lives in [microsoft/testfx](https://github.com/microsoft/testfx/tree/main/src/Platform/Microsoft.Testing.Platform) repository, and comes bundled with MSTest in 3.2.0-preview. This preview is available on [test-tools NuGet feed](https://pkgs.dev.azure.com/dnceng/public/_packaging/test-tools/nuget/v3/index.json).
+
+## Enabling MSTest Runner in an MSTest project
```

Comment:
Shall it be `a MSTest` instead of `an MSTest`?

Reply:
```suggestion
## Enabling MSTest Runner in a MSTest project
```